include/zephyr/bluetooth/bluetooth.h

```diff
@@ -2849,7 +2857,9 @@ int bt_le_filter_accept_list_clear(void);
 /**
  * @brief Set (LE) channel map.
  *
- * @param chan_map Channel map.
+ * Used to inform the Controller of which RF channels to use for LE communication with the Host.
```

Comment:
Not sure if this is correct. This is for the host to inform the controller about known channel classifications as far as I understand. The host can tell the controller which channels are bad and which are unknown, but it cannot tell the controller which ones to use. 

Possible also document that this shall not be sent twice within a 1-second period: 
![image](https://github.com/user-attachments/assets/85c59239-127d-48e8-b09c-70ef421f89cd)

Reply:
I'll add a note in regards to the one-second interval, though there is nothing (at least in the host AFAIS) enforcing this.